doc/setup/quickstart_cmake/index.md

```diff
@@ -207,3 +209,8 @@ For example, after adding `another_program.cpp` your `CMakeLists.txt` may look l
   export PATH=/usr/bin:$PATH
   ```
   and run `cmake` again.
+
+* **SDSL library not found**: `The SDSL library is required, but wasn't found.`
+  This has two potential sources:
+  + The repository wasn't cloned correctly. This can be verified by checking if `submodules/sdsl-lite/include/version.hpp` exists.
+  + Or, you are using MacOS and forgot to set `CC` and `CXX` variables and cmake is using Apple Clang as a compiler instead of GCC.
```

Comment:
```suggestion
* **SDSL library not found**: `The SDSL library is required, but wasn't found.`
  This has two potential sources:
  + The repository was not cloned correctly. This can be verified by checking whether `submodules/sdsl-lite/include/version.hpp` exists.
  + You are using macOS and did not set a C++ compiler for CMake. In this case, CMake may be using Apple Clang instead of GCC. See Compiling and Running on how to set a compiler for CMake.
``